### PR TITLE
Prelim products (proseco-only) review

### PR DESCRIPTION
**Pull request by taldcroft**
@jeanconn : this is a super-simple initial cut which at least makes a plot of every observation.  I was using in ipython and then looking at them in Mac preview.
```
run review
acas = get_acas('jan2819.pkl')
make_starcat_plots(acas, 'jan2819')
```

For JAN2819 prelim, I didn't see any obvious constellation problems, but several ERs have stars that are probably too close to the edge.  Should be straightforward to replicate the starcheck edge checks here, print the star ids and supply those ids to James for `exclude_ids_guide`.

### CODE REVIEW

File: proseco/preview.py [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+        for entry in self:
+            self.check_position_on_ccd(entry)
+
+        self.check_guide_geometry()
+        self.check_acq_p2()
+
+    def check_guide_geometry(self):
+        """Check for guide stars too tightly clustered
+
+        (1) Check for any set of n_guide-2 stars within 500" of each other.
```
Comment (jeanconn) [inline proseco/preview.py]: The n-2 check seems a fine place to start.  Not sure how to figure out a reasonable tolerance but 500" doesn't seem too crazy.

**Comment by jeanconn** [comment]
For ERs, I'd also like a check that there are 3 brighter than 9, which I can add.

**Comment by jeanconn** [comment]
Also @taldcroft, overall this prelim review tool looks great and is instantly helpful.  Thanks for working on it this weekend.

**Comment by taldcroft** [comment]
Superceded by `aca_preview` (https://github.com/sot/aca_preview/pull/1)

**Pull request closed without merging**